source/educators/concepts/accessibility/index.rst

```diff
@@ -0,0 +1,24 @@
+.. _Accessibility:
+
+#############################
+Accessibility
```

Comment:
Nit: I would call this something like "Accessibility in Course Creation" to distinguish it from the pages on developing an accessible product. If you agree I can change later on